src/main/scala/com/target/data_validator/validator/ColumnSumCheck.scala

```diff
@@ -63,6 +63,7 @@ case class ColumnSumCheck(
       case LongType => evaluate(r.getLong(idx))
       case FloatType => evaluate(r.getFloat(idx))
       case DoubleType => evaluate(r.getDouble(idx))
+      case ByteType => evaluate(r.getByte(idx))
       case ut => throw new Exception(s"Unsupported type for $name found in schema: $ut")
```

Comment:
This behaviour of throwing exception for unsupported type is different from columnMaxCheck, where it is treated as a normal check failure. I think we should make both of them consistent.

Reply:
I opened https://github.com/target/data-validator/issues/46 to handle that enhancement.